quickannotator/api/v1/annotation/utils.py

```diff
@@ -32,6 +73,43 @@ def get_progress(self) -> float:
         with self.lock:  # Lock when accessing progress
             return (self.progress / self.total) * 100
 
+@ray.remote(max_concurrency=2)  # Add max_concurrency=2
+class AnnotationImporter():
+    def __init__(self):
+        self.logger = LoggingManager.init_logger(constants.LoggerNames.RAY.value)
+
+    def import_from_tsv_row(self, project_id, image_path_col_name, data, columns):
+        # get slide path
+        slide_path = data[image_path_col_name].strip()
+        if (constants.TSVFields.FILE_PATH.value in columns) and (data[constants.TSVFields.FILE_PATH.value].strip()):
+                slide_path = data[constants.TSVFields.FILE_PATH.value].strip()        
+        # create slide if the slide path exist
+        if os.path.exists(fsmanager.nas_read.relative_to_global(slide_path)) is False:
+            self.logger.error(f"Slide path - {slide_path} not found")
+            raise Exception(f"Slide path - {slide_path} not found")
+        # create the image      
+        # if os.path.exists(fsmanager.nas_read.relative_to_global(slide_path)):
+        new_image = add_image_by_path(project_id, slide_path)
+        image_id = new_image.id
+
+        # Filter annotation classes ending with '_annotations'
+        annto_class_names = [col for col in columns if col.endswith(constants.ANNOTATION_CLASS_SUFFIX)]
+        for name in annto_class_names:
+            class_name = name[:-len(constants.ANNOTATION_CLASS_SUFFIX)]
+            print(new_image.name)
+            print(class_name)
+            self.logger.info(class_name)
```

Comment:
Can we log something more informative here?

Reply:
Sorry, these are my debug code. These need to be removed.

Reply:
Okay, I don't see any changes made here?